mpqp/core/instruction/gates/native_gates.py

```diff
@@ -413,6 +529,10 @@ def qiskit_gate(cls):
 
         return ZGate
 
+    @classproperty
```

Comment:
for this I think you could have `qasm2_gate = "z"` instead of a `classproperty`, we used class properties on the other ones to avoid the import in the class directly, because this would affect the library load time

Reply:
done